src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -148,13 +149,21 @@ func GetState(c Client, nodeID uint) (*NodeState, error) {
 	if err != nil {
 		return nil, err
 	}
+	hostAdminPort, err := getHostPort(
+		config.DefaultAdminPort,
+		containerJSON,
+	)
```

Comment:
If the status is `exited` the port will be 0:

```
$ rpk container status
  NODE-ID  STATUS   KAFKA-ADDRESS    ADMIN-ADDRESS    
  0        running  127.0.0.1:42221  127.0.0.1:39409  
  1        exited   127.0.0.1:0      127.0.0.1:0      
  2        running  127.0.0.1:35759  127.0.0.1:45435  
  
You can use rpk to interact with this cluster. E.g:

	rpk cluster info --brokers 127.0.0.1:42221,127.0.0.1:35759

You may also set an environment variable with the comma-separated list of
broker addresses:

	export REDPANDA_BROKERS="127.0.0.1:42221,127.0.0.1:35759"
	rpk cluster info
```

Maybe we should avoid printing the addresses if the status is != 'running'

Reply:
Just saw that this make `status` look weird, because it'll show an empty existing cluster. Instead, I think we can print the exited line and then have no addresses.